### PR TITLE
added signup page

### PR DESCRIPTION
**Pull request by athar05**
Added Signup Page

image

### CODE REVIEW

File: signup.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+<body class="page-container signup">
+  <!-- header starts -->
+  <div class="header">
+    <div class="nav">
+      <div class="nav-left">
+        <img src="./images/running-shoes.png" alt="">
+        <input class="search-bar" type="text" placeholder="Search..">
+        <a href="./login.html">
+          <div class="button button-icon"> <span> Login </span>
```
Comment (sashaboi) [inline signup.html:28]: some sort of vertical alignment might me required here for the icon and text, display flex, align items center might work

**Comment by sashaboi** [comment]
Once the alignment is fixed, I think this is ready to go

**Review by sashaboi — APPROVED** [review]
Once the alignment is fixed , this is ready to go !
Onkar Deshpande

**Review by krgarima — COMMENTED** [review]
The signup form should be aligned at the center. Give some width and margin (like width: 100px;
margin: auto;). Also, reduce it's height.

**Pull request merged**